experimental/packages/opentelemetry-sdk-node/test/util/resource-assertions.ts

```diff
@@ -317,14 +322,17 @@ const assertHasOneLabel = (prefix: string, resource: Resource): void => {
 
   assert.ok(
     hasOne,
-    'Resource must have one of the following attributes: ' +
+    'Must have one node Resource(s) starting with [' +
+      prefix +
+      '] matching the following attributes: ' +
       Object.entries(SemanticResourceAttributes)
         .reduce((result, [key, value]) => {
           if (key.startsWith(prefix)) {
             result.push(value);
           }
           return result;
         })
-        .join(', ')
+        .join(', ') +
+      JSON.stringify(Object.keys(SemanticResourceAttributes))
```

Comment:
As above additional debugging context when the test was failing that was useful.